src/debug/ee/debugger.cpp

```diff
@@ -2085,6 +2103,22 @@ HRESULT Debugger::Startup(void)
 
         RaiseStartupNotification();
 
+        // Also initialize the AppDomainEnumerationIPCBlock
```

Comment:
@mikem8361 Notice that `FEATURE_DBGIPC_TRANSPORT_VM` block above uses `m_pAppDomainCB` - that is initialized too late now.

Reply:
My change that is being reverted moved the m_pAppDomainCB initialization to almost the beginning of Debugger::Startup() so it should be initialized.

The change is moving the m_AppDomainCB back to where it was (later in Startup()).

Am I missing something?

Reply:
Sorry, I go it backward ... you are right. But I do think that the change of the order here caused the regression.

Reply:
And it is likely that we have the same regression on other Linux flavors and just do not see it as often.